### DISCUSSION TITLE
暗夜模式下tree控件的checked和selected样式问题

### DISCUSSION THREAD

**Discussion by fhmsha** [vbenjs/vue-vben-admin#475] [General]
![image](https://user-images.githubusercontent.com/7347071/114419422-fed32a80-9be5-11eb-808a-c31c65a812a4.png)

**Comment by 2yllll** [comment]
checkbox问题：去掉组件下 ，dark模式的多余border样式 
选中字体颜色：dark模式下，覆盖已选中tree-title的颜色
